### ISSUE TITLE
False positive for Z121 when using `_` for unused var names

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#533]
# Bug report

## What's wrong

This code triggers `Z121`:

```python
def logo_and_user():
    user, _ = User.objects.get_or_create(username='sobolevn')
    logo, _ = Logo.objects.get_or_create(username='sobolevn')  # here
```

Log: `test.py:4:11: Z121 Found usage of a variable marked as unused: _`
But it should not, since these variables are unused.

## How is that should be

It should not raise any `Z121` violations here.

## System information

I am using `8.0.1`

**Comment by sobolevn** [comment]
Closed by accident
